From d136c6dc1ef46900488df20bb842909b73479f74 Mon Sep 17 00:00:00 2001
From: Siddhesh Poyarekar <siddhesh@redhat.com>
Date: Thu, 5 Dec 2013 09:34:58 +0530
Subject: [PATCH] BZ #15941: Fix INSTALL file regeneration failure with
 makeinfo 5.x

I have skipped regenerating the INSTALL file because we want to
continue using the 4.x generator for now.
---
 Makefile                  |  4 ++--
 manual/install-plain.texi |  5 +++++
 manual/install.texi       |  6 +++---
 5 files changed, 24 insertions(+), 10 deletions(-)
 create mode 100644 manual/install-plain.texi

diff --git a/Makefile b/Makefile
index ccba09a..1998756 100644
--- a/Makefile
+++ b/Makefile
@@ -377,8 +377,8 @@ dist: dist-prepare
 	fi
 endif
 
-INSTALL: manual/install.texi manual/macros.texi \
-	 $(common-objpfx)manual/pkgvers.texi
+INSTALL: manual/install-plain.texi manual/macros.texi \
+	 $(common-objpfx)manual/pkgvers.texi manual/install.texi
 	makeinfo --no-validate --plaintext --no-number-sections \
 		 -I$(common-objpfx)manual $< -o $@-tmp
 	$(AWK) 'NF == 0 { ++n; next } \
diff --git a/manual/install-plain.texi b/manual/install-plain.texi
new file mode 100644
index 0000000..c5179e7
--- /dev/null
+++ b/manual/install-plain.texi
@@ -0,0 +1,5 @@
+@c This is for making the `INSTALL' file for the distribution.
+@c Makeinfo ignores it when processing the file from the include.
+@setfilename INSTALL
+@set plain
+@include install.texi
diff --git a/manual/install.texi b/manual/install.texi
index aaf27c9..535dc19 100644
--- a/manual/install.texi
+++ b/manual/install.texi
@@ -1,10 +1,10 @@
-@c This is for making the `INSTALL' file for the distribution.
-@c Makeinfo ignores it when processing the file from the include.
-@setfilename INSTALL
 @include macros.texi
 @include pkgvers.texi
 
+@ifclear plain
 @node Installation, Maintenance, Library Summary, Top
+@end ifclear
+
 @c %MENU% How to install the GNU C Library
 @appendix Installing @theglibc{}
 
-- 
1.8.5.3

